kiwix-build.py

```diff
@@ -399,7 +400,14 @@ def download(self, what, where=None):
             if what.sha256 == get_sha256(file_path):
                 raise SkipCommand()
             os.remove(file_path)
-        urllib.request.urlretrieve(file_url, file_path)
+        if options.no_cert_check == True:
+            context = ssl.create_default_context()
+            context.check_hostname = False
+            context.verify_mode = ssl.CERT_NONE
+            with urllib.request.urlopen(file_url, context=context) as resource, open(file_path, 'wb') as file:
+                file.write(resource.read())
+        else:
+            urllib.request.urlretrieve(file_url, file_path)
```

Comment:
I would prefer to have only "way" (line) to effectively download the file (and so reduce different code to do the same thing)

Something like that would be better:
```python
if options.no_cert_check == True:
    # Set option to not check certificate
    context = ...
else:
    # Set option to check certificate
    context = ...
with urllib.request.urlopen(file_url, context=context) as resource, open(file_path, 'wb') as file:
    # Write resource data into file.
    ...
```